clients/cpp-console/include/DialogManager.h

```diff
@@ -47,6 +47,9 @@ class DialogManager
     void PauseKws();
     void StartKws();
     void StartListening();
+    void StopListening();
```

Comment:
PTS stops playback and resets dialog state by reconnecting. It's not really about stopping listening (since if KWS was on before, you turn it back on after). I would simply call this Stop()

Reply:
Updated.